app/javascript/article-form/articleForm.jsx

```diff
@@ -60,12 +60,12 @@ export class ArticleForm extends Component {
   static propTypes = {
     version: PropTypes.string.isRequired,
     article: PropTypes.string.isRequired,
-    organizations: PropTypes.string,
+    organizations: PropTypes.oneOfType([PropTypes.array, PropTypes.string]),
```

Comment:
are we sure it can be a string? I followed `organizations` in this file down the rabbit hole of components and I arrived at 

https://github.com/forem/forem/blob/4cb7fd87016616ffe689c7d829d113a13b1f6b90/app/javascript/organization/OrganizationPicker.jsx#L63

in the `OrganizationPicker` component